### DISCUSSION TITLE
error: not all models converging when using run_assessment function

### DISCUSSION THREAD

**Discussion by NadiaFacciola** [osparcomm/HARSAT#428] [General]
When using the run_assessment function, I kept encountering issues where some of the models were converging but many of them were not. I found that it was due to an issue with the Matrix package. I uninstalled all the Matrix packages (I had two in my library) and re-installed the latest version of Matrix and the converging issue went away.

**Comment by morungos** [comment]
See also #394, which discusses a very different problem but which might well be related. 

We've seen issues with the `Matrix` binary incompatibility before, and I will not be surprised if we see it again. I'm not entirely sure how we can either detect it or handle it, because it is deep down in dependencies.
